docs/ABOUT.md

```diff
@@ -1,21 +1,22 @@
-Dart brings object-oriented and typing together with the flexibility that JavaScript is known for.  It is designed to simplify common programming tasks. This has made the biggest impact in regards to web development via AngularDart which has the Angular framework implemented using Dart.
+Dart brings object-oriented and typing together with the flexibility that JavaScript is known for.  It is designed to simplify common programming tasks. In recent years, it's popularity amongst the software development community has grown tremendously.
 
-To dig deeper into Dart, [click here](https://en.wikipedia.org/wiki/Dart_(programming_language)).
+To dig deeper into Dart, [click here](https://www.dartlang.org/guides/language/language-tour).
 
 The primary concepts baked into the making of the language were:
 
  - It is simple, object-oriented, and familiar.
    * The language derives its syntax from Java and JavaScript; as well as Smalltalk.
+ - It has a built-in concurrency model that is easy to learn and use.
  - It provides tools out of the box to help development efforts.
  - It has strong support via many standard libraries.
  - It is architecture-neutral and portable.
  - It executes with high performance for practical applications and not just benchmarks.
 
-Standardization:
+Dart is a very versitile language that can be used in a variety of different ways:
 
-If you need to write a web/enterprise app, or you need data persistence or you need to do cryptography, no problem!
-Dart is all you require!
+ - Dart code can be transpiled into JavaScript for [web development](https://webdev.dartlang.org/), typically using [AngularDart](https://webdev.dartlang.org/angular).
+ - [Application servers](https://www.dartlang.org/dart-vm) can be written using Dart.
+ - Cross-platform mobile application development can be achieved through a popular framework called [Flutter](https://flutter.io/).
+ - [Google Firebase](https://github.com/firebase/firebase-dart) APIs can be integrated into Dart apps using the libraries written by the Firebase team.
 
 There is a rich ecosystem springing up around Dart because the standard (built-in) libraries can be supplemented by open source libraries found on [Pub](https://pub.dartlang.org).
-
-And the community has a voice in the development of the language since there are many standard tools/practices that extend the [Dart specification](https://www.dartlang.org/guides/language/spec), thanks to the [Dart Enhancement Proposals](https://github.com/dart-lang/dart_enhancement_proposals) (DEP)!
```

Comment:
I would recommend keeping this as it's important to highlight the impact the community can have on the language's specification

Reply:
The Dart Team is actually no longer accepting DEPs for community proposals:

https://github.com/dart-lang/dart_enhancement_proposals/blob/master/README.md

Reply:
Wow didn't hear that. That makes me sad.